src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -633,14 +633,35 @@ private bool IsMouseInsideNodeOrPreview(Point mousePosition)
                     {
                         isInside = true;
                     }
-
                     return HitTestFilterBehavior.Stop;
                 },
                 ht => HitTestResultBehavior.Stop,
                 new PointHitTestParameters(mousePosition));
             return isInside;
         }
 
+        /// <summary>
+        /// Checks whether a mouse event occurred at a position matching the preview.
+        /// Alternatives attempted:
+        /// - PreviewControl.IsMouseOver => This is always false
+        /// - HitTest on NodeView => Anomalous region that skips right part of preview if larger than node
+        /// - HitTest on Preview => Bounds become irreversible larger than "mouse over area" after preview is expanded
+        /// </summary>
+        /// <param name="e">A mouse event</param>
+        /// <returns>Whether the mouse is over the preview or not</returns>
+        private bool IsMouseInsidePreview(MouseEventArgs e)
```

Comment:
How is this method different in functionality from the one with the similar name: `IsMouseInsideNodeOrPreview`

Reply:
Yes, that's right @aparajit-pratap . I know it's kinda strange for a method summary, but why it was needed was hard to explain otherwise.

Reply:
do we know why the `anomalous` region exists? 😉  💀